### DISCUSSION TITLE
snipe-it

### DISCUSSION THREAD

**Discussion by MARCO1801** [snipe/snipe-it#13148] [General]
porque no me la sale la pantalla de interfaz al colocar la ruta: http://localhost/snipe-it/
me sale esto: 
Index of /snipe-it
  | Parent Directory |   | - |  
  | CODE_OF_CONDUCT.md | 2023-04-07 12:47 | 3.2K |  
  | CONTRIBUTING.md | 2023-04-07 12:47 | 317 |  
  | Dockerfile | 2023-04-07 12:47 | 4.3K |  
  | Dockerfile.alpine | 2023-04-07 12:47 | 2.6K |  
  | Dockerfile.fpm-alpine | 2023-04-07 12:47 | 3.6K |  
  | LICENSE | 2023-04-07 12:47 | 34K |  
  | Procfile | 2023-04-07 12:47 | 57 |  
  | SECURITY.md | 2023-04-07 12:47 | 1.1K |  
  | TESTING.md | 2023-04-07 12:47 | 2.4K |  
  | Vagrantfile | 2023-04-07 12:47 | 4.5K |  
  | _config.yml | 2023-04-07 12:47 | 0 |  
  | ansible/ | 2023-06-11 19:13 | - |  
  | app.json | 2023-04-07 12:47 | 5.6K |  
  | app/ | 2023-06-11 19:12 | - |  
  | artisan | 2023-04-07 12:47 | 1.6K |  
  | bootstrap/ | 2023-06-11 19:12 | - |  
  | composer.json | 2023-04-07 12:47 | 3.7K |  
  | composer.lock | 2023-04-07 12:47 | 590K |  
  | config/ | 2023-06-11 19:12 | - |  
  | crowdin.yml | 2023-04-07 12:47 | 286 |  
  | database/ | 2023-06-11 19:12 | - |  
  | docker-compose.yml | 2023-04-07 12:47 | 755 |  
  | docker/ | 2023-06-11 19:12 | - |  
  | heroku/ | 2023-06-11 19:12 | - |  
  | install.sh | 2023-04-07 12:47 | 371 |  
  | package-lock.json | 2023-04-07 12:47 | 1.0M |  
  | package.json | 2023-04-07 12:47 | 1.8K |  
  | phpmd.xml | 2023-04-07 12:47 | 2.3K |  
  | phpstan.neon.dist | 2023-04-07 12:47 | 210 |  
  | phpstan.neon.example | 2023-04-07 12:47 | 265 |  
  | phpunit.xml | 2023-04-07 12:47 | 1.2K |  
  | psalm.xml | 2023-04-07 12:47 | 573 |  
  | public/ | 2023-04-07 12:47 | - |  
  | resources/ | 2023-06-11 19:13 | - |  
  | routes/ | 2023-06-11 19:13 | - |  
  | sample_csvs/ | 2023-06-11 19:13 | - |  
  | server.php | 2023-04-07 12:47 | 567 |  
  | snipeit.sh | 2023-04-07 12:47 | 28K |  
  | storage/ | 2023-06-11 19:13 | - |  
  | stubs/ | 2023-06-11 19:13 | - |  
  | tests/ | 2023-06-11 19:13 | - |  
  | upgrade.php | 2023-04-07 12:47 | 16K |  
  | vendor/ | 2023-06-11 19:29 | - |  
  | webpack.mix.js | 2023-04-07 12:47 | 7.2K |  

<br class="Apple-interchange-newline">
</body>
</html>
Index of /snipe-it
[ICO]	[Name](http://localhost/snipe-it/?C=N;O=D)	[Last modified](http://localhost/snipe-it/?C=M;O=A)	[Size](http://localhost/snipe-it/?C=S;O=A)	[Description](http://localhost/snipe-it/?C=D;O=A)
[PARENTDIR]	[Parent Directory](http://localhost/)	 	-	 
[   ]	[CODE_OF_CONDUCT.md](http://localhost/snipe-it/CODE_OF_CONDUCT.md)	2023-04-07 12:47	3.2K	 
[   ]	[CONTRIBUTING.md](http://localhost/snipe-it/CONTRIBUTING.md)	2023-04-07 12:47	317	 
[   ]	[Dockerfile](http://localhost/snipe-it/Dockerfile)	2023-04-07 12:47	4.3K	 
[   ]	[Dockerfile.alpine](http://localhost/snipe-it/Dockerfile.alpine)	2023-04-07 12:47	2.6K	 
[   ]	[Dockerfile.fpm-alpine](http://localhost/snipe-it/Dockerfile.fpm-alpine)	2023-04-07 12:47	3.6K	 
[   ]	[LICENSE](http://localhost/snipe-it/LICENSE)	2023-04-07 12:47	34K	 
[   ]	[Procfile](http://localhost/snipe-it/Procfile)	2023-04-07 12:47	57	 
[   ]	[SECURITY.md](http://localhost/snipe-it/SECURITY.md)	2023-04-07 12:47	1.1K	 
[   ]	[TESTING.md](http://localhost/snipe-it/TESTING.md)	2023-04-07 12:47	2.4K	 
[   ]	[Vagrantfile](http://localhost/snipe-it/Vagrantfile)	2023-04-07 12:47	4.5K	 
[   ]	[_config.yml](http://localhost/snipe-it/_config.yml)	2023-04-07 12:47	0	 
[DIR]	[ansible/](http://localhost/snipe-it/ansible/)	2023-06-11 19:13	-	 
[   ]	[app.json](http://localhost/snipe-it/app.json)	2023-04-07 12:47	5.6K	 
[DIR]	[app/](http://localhost/snipe-it/app/)	2023-06-11 19:12	-	 
[   ]	[artisan](http://localhost/snipe-it/artisan)	2023-04-07 12:47	1.6K	 
[DIR]	[bootstrap/](http://localhost/snipe-it/bootstrap/)	2023-06-11 19:12	-	 
[   ]	[composer.json](http://localhost/snipe-it/composer.json)	2023-04-07 12:47	3.7K	 
[   ]	[composer.lock](http://localhost/snipe-it/composer.lock)	2023-04-07 12:47	590K	 
[DIR]	[config/](http://localhost/snipe-it/config/)	2023-06-11 19:12	-	 
[   ]	[crowdin.yml](http://localhost/snipe-it/crowdin.yml)	2023-04-07 12:47	286	 
[DIR]	[database/](http://localhost/snipe-it/database/)	2023-06-11 19:12	-	 
[   ]	[docker-compose.yml](http://localhost/snipe-it/docker-compose.yml)	2023-04-07 12:47	755	 
[DIR]	[docker/](http://localhost/snipe-it/docker/)	2023-06-11 19:12	-	 
[DIR]	[heroku/](http://localhost/snipe-it/heroku/)	2023-06-11 19:12	-	 
[   ]	[install.sh](http://localhost/snipe-it/install.sh)	2023-04-07 12:47	371	 
[   ]	[package-lock.json](http://localhost/snipe-it/package-lock.json)	2023-04-07 12:47	1.0M	 
[   ]	[package.json](http://localhost/snipe-it/package.json)	2023-04-07 12:47	1.8K	 
[   ]	[phpmd.xml](http://localhost/snipe-it/phpmd.xml)	2023-04-07 12:47	2.3K	 
[   ]	[phpstan.neon.dist](http://localhost/snipe-it/phpstan.neon.dist)	2023-04-07 12:47	210	 
[   ]	[phpstan.neon.example](http://localhost/snipe-it/phpstan.neon.example)	2023-04-07 12:47	265	 
[   ]	[phpunit.xml](http://localhost/snipe-it/phpunit.xml)	2023-04-07 12:47	1.2K	 
[   ]	[psalm.xml](http://localhost/snipe-it/psalm.xml)	2023-04-07 12:47	573	 
[DIR]	[public/](http://localhost/snipe-it/public/)	2023-04-07 12:47	-	 
[DIR]	[resources/](http://localhost/snipe-it/resources/)	2023-06-11 19:13	-	 
[DIR]	[routes/](http://localhost/snipe-it/routes/)	2023-06-11 19:13	-	 
[DIR]	[sample_csvs/](http://localhost/snipe-it/sample_csvs/)	2023-06-11 19:13	-	 
[TXT]	[server.php](http://localhost/snipe-it/server.php)	2023-04-07 12:47	567	 
[   ]	[snipeit.sh](http://localhost/snipe-it/snipeit.sh)	2023-04-07 12:47	28K	 
[DIR]	[storage/](http://localhost/snipe-it/storage/)	2023-06-11 19:13	-	 
[DIR]	[stubs/](http://localhost/snipe-it/stubs/)	2023-06-11 19:13	-	 
[DIR]	[tests/](http://localhost/snipe-it/tests/)	2023-06-11 19:13	-	 
[TXT]	[upgrade.php](http://localhost/snipe-it/upgrade.php)	2023-04-07 12:47	16K	 
[DIR]	[vendor/](http://localhost/snipe-it/vendor/)	2023-06-11 19:29	-	 
[   ]	[webpack.mix.js](http://localhost/snipe-it/webpack.mix.js)	2023-04-07 12:47	7.2K

**Comment by marcusmoore** [comment]
No hablo español y espero que esto ayude.

Parece que su servidor web no está configurado correctamente. Si está utilizando Linux, consulte la documentación para obtener más ayuda: https://snipe-it.readme.io/docs/linuxosx
